persistence/src/main/java/cz/muni/fi/dao/TeamDaoImpl.java

```diff
@@ -19,23 +19,29 @@
     private EntityManager em;
 
     public void addTeam(Team team) {
-        if(team == null){
```

Comment:
Why are you changing teamImpl?

Reply:
because i told you in FB message im gonna do that